backend/.env.example

```
@@ -0,0 +1,28 @@
+# Google Service Account Configuration
+GOOGLE_SERVICE_ACCOUNT_TYPE=service_account
+GOOGLE_SERVICE_ACCOUNT_PROJECT_ID=multi-product-viewer
+GOOGLE_SERVICE_ACCOUNT_PRIVATE_KEY_ID=2418ec04c32e3e358d8cd4203fef114467fc7aff
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GOOGLE_SERVICE_ACCOUNT_PRIVATE_KEY_ID key should go before the GOOGLE_SERVICE_ACCOUNT_PROJECT_ID key

(UnorderedKey)